internal/api/entry.go

```diff
@@ -4,24 +4,39 @@
 package api // import "miniflux.app/v2/internal/api"
 
 import (
-	json_parser "encoding/json"
-	"errors"
-	"net/http"
-	"strconv"
-	"time"
-
-	"miniflux.app/v2/internal/config"
-	"miniflux.app/v2/internal/http/request"
-	"miniflux.app/v2/internal/http/response/json"
-	"miniflux.app/v2/internal/integration"
-	"miniflux.app/v2/internal/mediaproxy"
-	"miniflux.app/v2/internal/model"
-	"miniflux.app/v2/internal/reader/processor"
-	"miniflux.app/v2/internal/reader/readingtime"
-	"miniflux.app/v2/internal/storage"
-	"miniflux.app/v2/internal/validator"
+    json_parser "encoding/json"
+    "errors"
+    "net/http"
+    "strconv"
+    "time"
+
+    "miniflux.app/v2/internal/config"
+    "miniflux.app/v2/internal/crypto"
+    "miniflux.app/v2/internal/http/request"
+    "miniflux.app/v2/internal/http/response/json"
+    "miniflux.app/v2/internal/integration"
+    "miniflux.app/v2/internal/mediaproxy"
+    "miniflux.app/v2/internal/model"
+    "miniflux.app/v2/internal/reader/processor"
+    "miniflux.app/v2/internal/reader/readingtime"
+    "miniflux.app/v2/internal/storage"
+    "miniflux.app/v2/internal/validator"
 )
 
+// EntryImportRequest represents a manually imported entry for a feed.
+type EntryImportRequest struct {
```

Comment:
This struct should go in `payload.go` for consistency with the existing code base.